lib/ansible/modules/pip.py

```diff
@@ -120,6 +120,8 @@
     platform:
         platforms: posix
 notes:
+   - Python installations marked externally-managed (as defined by PEP668) cannot be updated by pip versions >= 23.0.1 without the use of
+     a virtual environment or setting the environment variable ``PIP_BREAK_SYSTEM_PACKAGES=1``.
```

Comment:
@nitzmahone or the `--break-system-packages` CLI flag for that matter. Is it intentional to omit this detail?

Reply:
The downside to the CLI flag is that it requires pip 23.0.1 or later. The environment variable will be ignored by pip versions that don't understand it. Using it allows the task to work regardless of pip version (except `pip==23.0.0`).